climada/util/alpha_shape.py

```diff
@@ -0,0 +1,99 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Compute the concave hull of a set of points.
+"""
+
+import numpy as np
+
+from shapely.ops import cascaded_union, polygonize
+from scipy.spatial import Delaunay
+import math
+import shapely.geometry as geometry
+import pylab as pl
+
+def alpha_shape(points, alpha):

```

Comment:
This function was blindly copied from another source.  Please at least adapt the docstrings.

Reply:
I'll update when we decide to keep it =)

Reply:
I've removed the alpha_shape and the hull_bruned_area method in commit 13a2c8c as calculating area burned is not used and the module is complex already

Reply:
Good. To be sure it does get spotted: It would be useful if you maybe could make a branch containing simply this module for potential future use.